### DISCUSSION TITLE
how theme "symbol 2" & "return to letters" buttons?

### DISCUSSION THREAD

**Discussion by Pun-e** [florisboard/florisboard#1341] [Q&A]
The theming guide is awesome, but lacks information on how to access a few of the special keys.

I tried obvious modifications like "view_letters" & "view_keyboard" but no dice

**Comment by patrickgold** [comment]
Have a look at the layouts in https://github.com/florisboard/florisboard/tree/master/app/src/main/assets/ime/text

For example in the modifier layout for symbol2, you can see that the switch to letters/symbols button is named `view_characters` / `view_symbols`:https://github.com/florisboard/florisboard/blob/1ef7656c6b4cc57287a57e74271c8cae3f23b9a2/app/src/main/assets/ime/text/symbols2/mod/%24default.json#L9-L14

With this you should be able to customize these buttons.

I am currently in parallel to the preference rework working on a new theme engine + customization UI, where themeing such buttons will be much easier, without having to look in the wiki or even searching for the key ID.

**Comment by blogmangit** [comment]
hi there read this wiki

https://github.com/florisboard/florisboard/wiki/Using-the-new-Theme-Editor

the symbol to alphabet & alphabet to symbol key is called as follows 

key:view_symbols

add as a group
